lucet-spectest/src/script.rs

```diff
@@ -70,6 +70,7 @@ impl ScriptEnv {
         let compiler = Compiler::new(
             module,
             OptLevel::default(),
+            CpuFeatures::baseline(),
```

Comment:
Interesting choice to be made here (and also with optimization level): this will end up testing Cranelift with codegen settings we'll almost never actually use in practice

Reply:
Definitely. Since we were talking about revisiting the spectest runner anyway, I'd _love_ to run spectests under at least both the minimum supported hardware, and most recent supported hardware.